vnext/Microsoft.ReactNative.Managed/IReactPromise.cs

```diff
@@ -30,24 +30,4 @@ public interface IReactPromise<T>
     // Report an Error.
     void Reject(ReactError error);
   }
-
-  public class ReactError
```

Comment:
>ReactError [](http://example.com/codeflow?start=15&length=10)

Thanks for factoring out class to separate file! 
Question: Should we do it more broadly?

Reply:
I think so. I guess it is one of the best practices. Do you think we should do it with attributes?